articles/techniques/testing-and-debugging.md

```diff
@@ -256,13 +256,13 @@ Notice how the IDs of the qubits are shown at the top in their significant order
 ***
 
 
-@"microsoft.quantum.extensions.diagnostics.dumpmachine" is part of the  <xref:microsoft.quantum.extensions.diagnostics> namespace, so in order to use it you must add an `open` statement:
+<xref:microsoft.quantum.extensions.diagnostics.dumpmachine> is part of the  <xref:microsoft.quantum.extensions.diagnostics> namespace, so in order to use it you must add an `open` statement:
 
 ```qsharp
 namespace Samples
 {
     open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Extensions.Diagnostics;
```

Comment:
It looks like there's been a regression as of c62c7c18438b969273d7d95bf52171f75abdf117, in that the correct namespace name is `Microsoft.Quantum.Diagnostics`.